websites/J/JavaScript/presence.ts

```diff
@@ -11,35 +11,19 @@ presence.on("UpdateData", async () => {
 
     if(paths[3] !== 'JavaScript')return;
 
-    if(paths[0] === 'es'){
-        if(!paths[4]){
-            data.details = `Viendo la pagina principal...`;
-        } else {
-            data.details = `Viendo ${paths[4]}`;
-            paths.splice(0, 1);
-            paths.splice(0, 1);
-            paths.splice(0, 1);
-            paths.splice(0, 1);
-            paths.splice(0, 1);
-            if(paths[0]){
-                data.state = `Tema: ${paths.join(', ')}`;
-            }
-        }
+    if(!paths[4]){
+        data.details = `Looking the main page...`;
     } else {
-        if(!paths[4]){
-            data.details = `Looking the main page...`;
-        } else {
-            data.details = `Looking at ${paths[4]}`;
-            paths.splice(0, 1);
-            paths.splice(0, 1);
-            paths.splice(0, 1);
-            paths.splice(0, 1);
-            paths.splice(0, 1);
-            if(paths[0]){
-                data.state = `Topic: ${paths.join(', ')}`;
-            }
+        data.details = `Looking at ${paths[4]}`;
+        paths.splice(0, 1);
+        paths.splice(0, 1);
+        paths.splice(0, 1);
+        paths.splice(0, 1);
+        paths.splice(0, 1);
```

Comment:
what is up with all the splicing?

Reply:
with that i remove this thing "es/docs/Web/JavaScript"

Reply:
You can do this in different way. ._. You don't need to even splice it up but 'kay